xsd/netex_framework/netex_reusableComponents/netex_facility_support.xsd

```diff
@@ -672,7 +672,7 @@ Rail transport, Roads and Road transport
 			<xsd:enumeration value="internet"/>
 			<xsd:enumeration value="postOffice"/>
 			<xsd:enumeration value="postBox"/>
-		<xsd:enumeration value="UsbPowerSocket">
+			<xsd:enumeration value="UsbPowerSocket">
```

Comment:
```suggestion
			<xsd:enumeration value="usbPowerSocket">
```
Please - especially with enumerations - make sure you follow the conventions. It is crazy difficult to convince Christoph and Ulf to fix typo's later.